### PR TITLE
refactor(ci): Lint PR diffs only

### PR DESCRIPTION
**Pull request by ivov**

### CODE REVIEW

File: .github/workflows/ci-pull-requests.yml [hunk 1]
```diff
@@ -13,18 +13,27 @@ jobs:
...
+
+      - name: Test
+        run: npm run test
+
+      - name: Fetch base branch for `git diff`
+        run: git fetch origin ${{ github.event.pull_request.base.ref }}:${{ github.event.pull_request.base.ref }}
+
+      - name: Run ESLint on changes only
         env:
-          CI: true
```
Comment (netroy) [inline .github/workflows/ci-pull-requests.yml:30]: I think `CI=true` is used by npm to ensure reproducible builds. It also disables npm telemetrry, and auditing (I think).

**Review by netroy — APPROVED** [review]
besides the missing `CI: true`, LGTM

File: .github/workflows/ci-pull-requests.yml (hunk 1 above)
Comment (ivov) [reply .github/workflows/ci-pull-requests.yml:30]: I was going off of this: https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/

Set anyway?

Comment (netroy) [reply .github/workflows/ci-pull-requests.yml:30]: wasn't aware of that. makes sense.

**Comment by janober** [comment]
Got released with `n8n@0.195.0`

**Pull request merged**